student.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯(ツ)/¯'
+  end
+end
+
+student = Student.new('univ')
+puts student.play_hooky
```

Comment:
[Optional] To prevent executing these lines of code when the file is imported, kindly remove these lines of code.